src/main/kotlin/no/skatteetaten/aurora/gobo/integration/phil/PhilService.kt

```diff
@@ -34,12 +34,31 @@ class PhilServiceReactive(
             }
             .bodyToMono<List<DeploymentResource>>()
             .awaitFirstOrNull()
+
+    override suspend fun deleteEnvironment(environment: String, token: String) =
+        webClient
+            .delete()
+            .uri("/environments/$environment")
+            .header(HttpHeaders.AUTHORIZATION, "Bearer $token")
+            .retrieve()
+            .onStatusNotOk { status, body ->
+                throw PhilIntegrationException(
+                    message = "Request failed when deploying environment",
+                    integrationResponse = body,
+                    status = status
+                )
+            }
+            .bodyToMono<List<DeletionResource>>()
+            .awaitFirstOrNull()
 }
 
 interface PhilService {
```

Comment:
usikker på hvorfor jeg ikke har fanget opp dette før, men denne bør nok ikke hete PhilService. Den ligger i `phil` package, men ressursen her er vel environment? Så noe sånt som `EnvironmentService`

Reply:
Ser den. Tror initiell tanke var at siden dette var den eneste tjenesten som gikk mot phil, så valgte jeg det enkle navnet. Er selvsagt enig i at det bør navnes etter ressursen som behandles, og at dette var litt for lettvint.  Renavner.

Reply:
Har renavnet til EnvironmentService.  Det gir en bitteliten mulighet for navneforvirring med ...booober.EnvironmentService , men det går nok bra.